include/labwc.h

```diff
@@ -254,8 +254,16 @@ struct server {
 
 	/* Tree for all non-layer xdg/xwayland-shell surfaces */
 	struct wlr_scene_tree *view_tree;
-	/* Tree for all non-layer xdg/xwayland-shell surfaces with always-on-top */
+
+	/*
+	 * Popups need to be rendered above always-on-top views, so we reparent
+	 * them to this dedicated tree
+	 */
+	struct wlr_scene_tree *popup_tree;
```

Comment:
Should we name this `server->xdg_popup_tree`?

Reply:
Done